modules/tools/atlantis/main.tf

```diff
@@ -0,0 +1,55 @@
+resource "helm_release" "atlantis" {
+  chart      = "atlantis"
+  name       = "atlantis"
+  repository = "https://runatlantis.github.io/helm-charts"
+  version    = "3.14.2"
+  namespace  = var.namespace
+
+  values = [
+    templatefile("${path.module}/values.yaml.tpl", {
+      default_terraform_version = var.default_terraform_version
+      role_arn                  = var.role_arn
+      ingress_hostname          = var.ingress_private_hostname
+      ingress_class             = var.ingress_private_class
+    })
+  ]
+
+  set_sensitive {
+    name  = "github.token"
+    value = var.github_token
+  }
+
+  set_sensitive {
+    name  = "github.secret"
+    value = var.github_webhook_secret
+  }
+
+  set_sensitive {
+    name  = "github.user"
+    value = var.github_username
+  }
+}
+
+resource "kubernetes_ingress" "public_events_webhook" {
```

Comment:
the `/events` endpoint used via GitHub webhook (which is secured with a secret) needs to be public, so this ingress is created for only that purpose

Reply:
the helm chart creates an ingress for the dashboard, which uses the internal ingress controller